olive/systems/azureml/aml_system.py

```diff
@@ -21,13 +21,23 @@
 from olive.constants import Framework
 from olive.evaluator.metric import Metric, MetricResult
 from olive.hardware.accelerator import AcceleratorSpec
-from olive.model import ModelConfig, ModelStorageKind, OliveModel, ONNXModel
+from olive.model import ModelConfig, OliveModel
 from olive.passes.olive_pass import Pass
+from olive.resource_path import AZUREML_RESOURCE_TYPES, AzureMLModelConfig, ResourcePath, ResourceType
 from olive.systems.common import AzureMLDockerConfig, SystemType
 from olive.systems.olive_system import OliveSystem
 
 logger = logging.getLogger(__name__)
 
+RESOURCE_TYPE_TO_ASSET_TYPE = {
+    ResourceType.LocalFile: AssetTypes.URI_FILE,
+    ResourceType.LocalFolder: AssetTypes.URI_FOLDER,
+    ResourceType.StringName: AssetTypes.URI_FILE,  # this is a place holder since we won't upload a string as input
```

Comment:
Could we set the value of `StringName` as None to stand for place holder? Or just remove this line?

Reply:
In methods like `create_model_inputs`, it uses the mapped values so `None` is not accepted.

Reply:
Ic, you may do not want to add too many condition when create the model input. 

Then it is fine.